### PR TITLE
chore: WIP: Add DpModal to Storybook

### PR DESCRIPTION
**Pull request by ahmad-demos**
- WIP: Add DpModal to Storybook
- Fix getComputedStyle error `Window.getComputedStyle: Argument 1 does not implement interface Element.`, adjust DpModal Storybook.

### CODE REVIEW

File: src/components/DpModal/DpModal.stories.mdx [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+The Modal component allows users to open and close a modal dialog or popup by clicking a button, link
+or performing a specific action.
+
+
+## Custom usage
+
+```html
+<dp-modal id="openModal" ref="openModal" v-bind="args">
+  <template v-slot:header>
+    Header Title
```
Comment (spiess-demos) [inline src/components/DpModal/DpModal.stories.mdx:20]:
```suggestion
    Short, descriptive modal title
```

File: src/components/DpModal/DpModal.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+            <dp-modal id="openModal" ref="openModal" v-bind="args">
+              <template v-slot:header>
+                Header Title
+              </template>
+              <template>
+                <p>
+                  Template Code & Text
+                </p>
+              </template>
+            </dp-modal>
```
Comment (spiess-demos) [inline src/components/DpModal/DpModal.stories.tsx:32]: We could apply the same changes here that i suggested above.

File: src/components/DpModal/DpModal.stories.mdx [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+# Modal
+
+The Modal component allows users to open and close a modal dialog or popup by clicking a button, link
+or performing a specific action.
+
+
+## Custom usage
+
+```html
+<dp-modal id="openModal" ref="openModal" v-bind="args">
```
[the remark this answers is not in the document]
Comment (ahmad-demos) [reply src/components/DpModal/DpModal.stories.mdx]: Done in https://github.com/demos-europe/demosplan-ui/pull/538/commits/11ce140eabcfc7067479d1c24cb36c95236ae48f

File: src/components/DpModal/DpModal.stories.tsx (hunk 2 above)
Comment (ahmad-demos) [reply src/components/DpModal/DpModal.stories.tsx]: Done in https://github.com/demos-europe/demosplan-ui/pull/538/commits/11ce140eabcfc7067479d1c24cb36c95236ae48f

File: src/components/DpModal/DpModal.stories.tsx [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+                @click.stop.prevent="$refs.openModal.toggle()"
+                role="button">
+              Open Modal
+            </a>
+            <dp-modal
+                id="openModal"
+                ref="openModal"
+                v-bind="args">
+              <template v-slot:header>
+                Header Title
```
Comment (spiess-demos) [inline src/components/DpModal/DpModal.stories.tsx:28]:
```suggestion
                Short, descriptive modal title
```

File: src/components/DpModal/DpModal.stories.tsx [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+              </template>
+              <template>
+                <p>
+                  Template Code & Text
```
Comment (spiess-demos) [inline src/components/DpModal/DpModal.stories.tsx:32]:
```suggestion
                  Modal content
```

**Review by spiess-demos — CHANGES_REQUESTED** [review]

**Review by spiess-demos — APPROVED** [review]
Lgtm!

**Pull request merged**